cypress/integration/frontsite/literatureRequest.test.spac.js

```diff
@@ -0,0 +1,30 @@
+describe('literature request form', () => {
+  Cypress.Cookies.defaults({
+    preserve: 'csrftoken',
+  });
+
+  it('should redirect to login if not logged in', () => {
+    cy.shouldRedirectToLogin('/request');
+  });
+
+  it('should display form whe logged in', () => {
+    cy.login({ email: 'patron1@test.ch', password: '123456' });
+    cy.visit('/request');
+    cy.url().should('eq', Cypress.config().baseUrl + '/request');
+    cy.contains('h1', 'Request new literature');
+  });
+
+  it('should submit form', () => {
+    cy.login({ email: 'patron1@test.ch', password: '123456' });
+    cy.visit('/request');
+
+    cy.get('input#title').type('A new book');
+    cy.get('input#medium').click({ force: true });
```

Comment:
Doesn't it work without `force`?
I think it's discouraged to use this parameter (if possible) because it bypasses the visibility/obstruction checks. For instance the <kbd>X</kbd> button in form arrays is currently unclickable because 'hidden' by another element; and we really want to catch these kinds of bugs.